crates/modelardb_compression/src/types.rs

```diff
@@ -750,7 +726,7 @@ mod tests {
         );
     }
 
-    fn test_encoding_decoding_for_swing(
+    fn assert_encoding_decoding_for_swing(
```

Comment:
Maybe rename to something like `assert_encoding_decoding_valid_for_swing`.

Reply:
Fixed.